source/_integrations/zwave_js.markdown

```diff
@@ -35,43 +35,20 @@ ha_platforms:
   - switch
 ---
 
-This integration allows you to control a Z-Wave network via the [Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/) driver. This is our recommended Z-Wave integration for Home Assistant.
+This integration allows you to control a Z-Wave network via the [Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/) driver. This is our recommended Z-Wave integration for Home Assistant. If you're new to Home Assistant, use Z-Wave JS. The `zwave` and `ozw` integrations have been marked as deprecated and will no longer receive any updates. See [our migration instructions](#migrating-from-previous-z-wave-implementations) to migrate from a prior integration.
```

Comment:
This page is specifically towards Z-Wave JS, do we need to mention zwave/ozw at this point even?

The `zwave` and `ozw` pages have large alerts on their deprecation already.

Reply:
I just relocated that portion I think. I defer to whomever. We need a specific reference to the migration instructions at the top. That's all I care about.

Reply:
I agree, however, it isn't written from a migration focus, but from a "new Home Assistant" perspective. The line even starts with: "If you're new to Home Assistant", which is not the case for user migrating from `zwave` or `ozw`.

Reply:
Unresolving this one, as per my last comment.

I think we should rewrite this as something like:

"If you are currently using one of the deprecated `zwave` or `ozw` integrations, see [our migration instructions](#migrating-from-previous-z-wave-implementations) to migrate from a prior integration."

Reply:
That's nearly what I wrote verbatim? You're looking at the old one.